cds-snc/cppd-medical-report
# Add Overall Health route

# Description

Added the Overall Health step.

# Test Instructions

- With application running locally, visit `/en/health`
- Submit the form without filling it in to observe the validation
- Submit the completed form and it will take you to the confirmation screen (next step will be in the next PR)
